index.html

```diff
@@ -0,0 +1,33 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="utf-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Saul Greene Portfolio</title>
+    <link rel="stylesheet" href="style.css">
+    <link rel="icon" type="image/x-icon"
+      href="assets/sauls-face.png"
+    >
+    <link rel="stylesheet" href="reset.css">
```

Comment:
I get a 404 error in my browser console because your reset style sheet isn't linked correctly. Please change this to:

```
<link rel="stylesheet" href="vendor/reset.css">
```